client/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "axios": "^0.18.0",
+    "passport": "^0.4.0",
+    "react": "^16.6.3",
+    "react-datepicker": "^1.8.0",
+    "react-dom": "^16.6.3",
+    "react-fontawesome": "^1.6.1",
+    "react-router-dom": "^4.3.1",
+    "react-scripts": "2.1.1",
+    "react-select": "^2.1.1",
+    "react-validation": "^3.0.7",
+    "styled-components": "^4.1.1"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "browserslist": [
+    ">0.2%",
+    "not dead",
+    "not ie <= 11",
+    "not op_mini all"
+  ],
+  "proxy": "http://localhost:9000"
+}
```

Comment:
When you run your server tests, Jest will look inside 'client' for test files. You could add some Jest settings to this package JSON to exclude the 'client' folder.

Reply:
https://jestjs.io/docs/en/configuration#testpathignorepatterns-array-string

Reply:
I've set this up so it only looks into the test folder we want it to look in. Have created an initial test to ensure the test set up is working and all is passing.

Reply:
@arrested-developer  thank you, I think that  is useful comment